modules/geo-layers/src/tile-layer/utils.js

```diff
@@ -117,8 +117,8 @@ function getIndexingCoords(bbox, scale, modelMatrixInverse) {
   return bbox.map(i => (i * scale) / TILE_SIZE);
 }
 
-function getScale(z) {
-  return Math.pow(2, z);
+function getScale(z, tileSize = TILE_SIZE) {
```

Comment:
As an internal function, I recommend always explicitly setting the arguments for readability. Fallbacks may have unintended consequences.

Reply:
Ok yes, seems unnecessary anyway since it is only ever used where `tileSize` would (in theory) exist anyway.